linera-views/README.md

```diff
@@ -11,6 +11,134 @@ atomically persisted in storage).
 
 See the [CONTRIBUTING](../CONTRIBUTING.md) file for how to help out.
 
+## Design
+
+# Possible clients
+
+We designed a `KeyValueStoreClient` trait that implements the basic functionalities
+of a ke-value store whose keys are `Vec<u8>` and whose values are `Vec<u8>`.
+
+A number of key-value stores are supported by the system:
+* `memory` is using the memory (and uses internally a simple B-Tree map).
+* `RocksDB` is a disk based key-value store
+* `DynamoDb` is the AWS based DynamoDb service.
+
+For each of those we provide a type that implements the trait.
```

Comment:
Maybe move this sentence up and replace the one before the enumeration:
`We provide an implementation of the trait `KeyValueStoreClient` for the following key-value stores:`

Reply:
ok.